src/pages/WishlistPage/WishlistPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useContext,useEffect } from 'react';
+import ProductCard from '../../components/ProductCard/ProductCard';
+import { StateContext } from '../../Context';
+import './WishlistPage.css';
+
+const WishlistPage = () => {
+    const {state,dispatch}=useContext(StateContext)
+    const encodedToken = localStorage.getItem('token')
```

Comment:
use from the useAuth that u have created

Reply:
Actually authentication is not yet implemented properly so I have not used useAuth here. It will be done after adding authentication. As of now just hard coded it.